### DISCUSSION TITLE
Blind SQLi fuzzing template

### DISCUSSION THREAD

**Discussion by nazmul-ethi** [projectdiscovery/nuclei#4100] [Q&A]
Hello!

Does the Nuclei tool have any Time-based SQLi template? Or is this possible to create a Time-based SQLi template?

**Comment by ehsandeep** [comment]
See https://templates.nuclei.sh/public/CVE-2022-1768; you can use `duration` dsl helper.

**Reply by nazmul-ethi** [reply]
Thanks, mam! Can you please give me a Time-based SQLi template FUZZING template?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
